errai-security/errai-security-picketlink/src/main/java/org/jboss/errai/security/server/PicketLinkBasicModelServices.java

```diff
@@ -0,0 +1,66 @@
+package org.jboss.errai.security.server;
+
+import org.picketlink.idm.IdentityManagementException;
+import org.picketlink.idm.model.Account;
+import org.picketlink.idm.model.IdentityType;
+import org.picketlink.idm.model.basic.Group;
+import org.picketlink.idm.model.basic.Role;
+
+public interface PicketLinkBasicModelServices {
```

Comment:
Since BasicModel class from PicketLink has all method static, I created this interface to be able of mocking it in the Tests.